pkg/kubelet/pleg/generic.go

```diff
@@ -358,24 +357,6 @@ func (g *GenericPLEG) getPodIPs(pid types.UID, status *kubecontainer.PodStatus)
 		return nil
 	}
 
```

Comment:
`|| len(oldStatus.IPs) == 0 ` is unnecessary here

Reply:
done, thanks